bettersigntool/bettersigntool/SignCommand.cs

```diff
@@ -346,11 +344,8 @@ private bool DoSign(string filename)
 
             do
             {
-                if (attempt > 1)
```

Comment:
Without this if Isn't it going to immediately sleep for 3 seconds before even attempting to sign ?

Reply:
agreed

Reply:
It totally is. Not only did I miss that, so did my pre-PR reviewer ;)